backstop/tests/text-field.html

```diff
@@ -29,6 +29,17 @@
         flex-direction: column;
         gap: 11px;
       }
+
+      #block-labels { display: grid; }
```

Comment:
Do you plan to clean up these head styles? (could remove `display: grid` from #block-labels and could add `:not(:last-child)` to the margin-bottom in the next line, though you might need to add a `<br />` to separate the block and inline labels)

I only added these rules to demonstrate what was possible. In this HTML page, the emphasis should be on the styling more than the layout. And then you can have proper grid layouts in your info panel PR.